### PR TITLE
Peak of unimodal list DNC algorithm

### PR DESCRIPTION
**Pull request by choyiny**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: divide_and_conquer/peak.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    elif three[0] < three[2]:
+        if len(lst[:m]) == 2:
+            m -= 1
+        return peak(lst[m:])
+
+    # decreasing
+    else:
+        if len(lst[:m]) == 2:
+            m += 1
+        return peak(lst[:m])
```
Comment (xcodz-dot) [inline divide_and_conquer/peak.py:47]:
```suggestion
        return peak(lst[:m])

```
PEP 8 fix

Comment (choyiny) [reply divide_and_conquer/peak.py:47]: If you mean adding a newline to the end of file, it is already done. Github doesn't show line 48 in the `files` tab. Only when you click on raw you would see it.

Comment (xcodz-dot) [reply divide_and_conquer/peak.py:47]: whoops, I did not know that (I joined github last month). Thanks for letting me know

**Comment by choyiny** [comment]
Looking to get a `hacktoberfest-accepted` tag if possible. @poyea

File: divide_and_conquer/peak.py (hunk 1 above)
Comment (poyea) [inline divide_and_conquer/peak.py:47]:
```suggestion
        return peak(lst[:m])

if __name__ == "__main__":
    import doctest

    doctest.testmod()
```

**Comment by xcodz-dot** [comment]
run `pip install black`
run `black peak.py` and then reupload peak.py

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Pull request merged**